### PR TITLE
Missing OpenBSD gcc patches.

### PR DESCRIPTION
**Pull request by vext01**
These are the missing patches for OpenBSD/gcc.

Cheers

### CODE REVIEW

File: patches/openbsd_gcc_patches/patch-gcc_opts_c [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- gcc/opts.c.orig	Thu Feb 11 14:44:27 2016
```
Comment (ltratt) [inline patches/openbsd_gcc_patches/patch-gcc_opts_c:1]: Is this one missing its `$OpenBSD` line?

File: patches/openbsd_gcc_patches/patch-gcc_gcc_config_opt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+--- gcc/config.gcc.orig	Thu Feb 11 14:26:19 2016
```
Comment (ltratt) [inline patches/openbsd_gcc_patches/patch-gcc_gcc_config_opt:1]: Is this one missing its `$OpenBSD` line?

Comment (vext01) [inline patches/openbsd_gcc_patches/patch-gcc_gcc_config_opt:1]: The patches without `$OpenBSD$` didn't appear in OpenBSD CVS. The patches I have devised are only the hunks relating to position independent code (PIC and PIE).

Comment (ltratt) [inline patches/openbsd_gcc_patches/patch-gcc_gcc_config_opt:1]: Aha, OK.

**Comment by ltratt** [comment]
I think this is ready to be merged, right?

**Comment by vext01** [comment]
This is all in #109. We have the option to either merge this, or as part of #109. Up to you.

**Comment by ltratt** [comment]
These patches aren't part of #109, so I suggest we merge this, then rebase #109, and continue.

**Comment by vext01** [comment]
I see. Sorry, TooManyOpenPRsException :P

Please merge.

**Pull request merged**